R/tm_g_association.R

```diff
@@ -305,26 +314,10 @@ srv_tm_g_association <- function(id,
       # association
       ref_class_cov <- ifelse(association, ref_class, "NULL")
 
-      teal.code::chunks_push(
-        id = "title_call",
-        expression = substitute(
-          expr = title <- new_title,
-          env = list(new_title = paste(
-            "Association",
-            ifelse(ref_class_cov == "NULL", "for", "between"),
-            paste(lapply(vars_names, function(x) {
-              if (is.numeric(ANL[[x]]) && log_transformation) {
-                varname_w_label(x, ANL, prefix = "Log of ")
-              } else {
-                varname_w_label(x, ANL)
-              }
-            }), collapse = " / "),
-            ifelse(ref_class_cov == "NULL", "", paste("and", ref_cl_lbl))
-          ))
-        )
-      )
```

Comment:
You have removed this expression. It should be included in the quosure

Reply:
Ok, will look into it

Reply:
fixed, the title is back ok